lib/entities/semantic-form-meta.js

```diff
@@ -0,0 +1,45 @@
+import { Graph, parse, serialize } from '../util/rdflib';
+
+export const META_FILE_MATCHER = '-meta.ttl';
+export const TAILORED_META_FILE_MATCHER = {
+  additions: '-additions-meta.ttl',
+};
+
+const GRAPHS = {
+  global: 'http://form-meta/global',
+  additions: 'http://form-meta/global',
+  removals: 'http://form-meta/global',
+  content: 'http://form-meta/content',
+};
+
+export class SemanticFormMeta {
+
+  constructor(sources) {
+    this.sources = sources;
+
+    /* Initialize store with meta TTL */
+    this.store = new Graph();
+    if (this.global) {
+      parse(this.global.content, this.store, {graph: GRAPHS.content});
+    }
+    if (this.tailored.additions) {
```

Comment:
(not needed for this PR) but a removal would just run before the additions, and remove the triples in the store?